models/klaviyo__person_flow.sql

```diff
@@ -0,0 +1,22 @@
+with person_flow as (
```

Comment:
I'm struggling to understand why we should break `klaviyo__person_flow` and `klaviyo__person_campaign` into two distinct models.  Wouldn't you want all of your aggregated person's data in one place?

Reply:
yeah....i'm unsure if people analyze campaigns and flows in isolation from each other or not